src/ProgressOnderwijsUtils/Html/HtmlAttribute.cs

```diff
@@ -85,16 +123,16 @@ public Enumerator(HtmlAttributes attrs)
             public HtmlAttribute Current
                 => attributes[pos];
 
-                object IEnumerator.Current
-                => attributes[pos];
+            object IEnumerator.Current
+                => Current;
 
             public void Dispose() { }
 
             public bool MoveNext()
                 => ++pos < count;
 
             public void Reset()
-                => pos = 0;
+                => pos = -1;

```

Comment:
Technically this is a bugfix, but this method is essentially never used (api misdesign), so it's not exactly relevant.

But in case somebody does use it....